packages/styles/src/components/spacing.scss

```diff
@@ -0,0 +1,203 @@
+@forward './../variables/options';
+
+@use 'sass:map';
+
+@use './../themes/bootstrap/core' as *;
+@use './../themes/bootstrap/utilities' as *;
+@use './../variables/spacing';
+@use './../variables/breakpoints';
+
+// Define Post spacing variables
+:root {
+  @each $size, $length in spacing.$spacers {
+    --post-size-#{$size}: #{$length};
+  }
+
+  @each $size, $length in spacing.$negative-spacers {
+    --post-size-#{$size}: #{$length};
+  }
+}
+
+// Update bootstrap utilities
+$utilities: map-merge(
```

Comment:
`$utilities` is already declared using `$spacers` and `$negative-spacers` variables (see [here](https://getbootstrap.com/docs/5.3/utilities/spacing/#utilities-api)). 

Since you override `$spacers` in packages/styles/src/variables/_spacing.scss you shouldn't have to override `$utilities` again here.